applications/GeoMechanicsApplication/tests/test_load_on_block_2d_no_damping/README.md

```diff
@@ -0,0 +1,23 @@
+# Load on block 2D no damping
+
+This test verifies that in a block of plane strain elements, the motion induced by instant application of a line load does not damp out.
+
+## Setup
+
+This test consists of a 4 3-noded elements (using the UPwSmallStrainElement2D3N class) of $1x1 \mathrm{[m]}$. A
```

Comment:
The rendered 1x1 now looks a bit strange due to the math $x$, I think either not having that as math expression, or using $1 \mathrm{x} 1$ (i.e. `$1 \mathrm{x} 1$`) would be better.

Reply:
done